File: 2_condition_and_loop/vowel_consonant.rb
# Write a Python program to check whether an alphabet is a vowel or consonant
#
#
#



def vowel_or_consonant
  print "Something's value: "
  value = gets.chomp

  value.each_char { |char| puts check_what_is char  }
end
  
def check_what_is(str)
  if str =~ /[aeiou]/
    is_a = "vowel"
  else
    is_a = "consonant"
  end
  "#{str} is a: #{is_a}"
end


vowel_or_consonant

File: ruby_and_python/2_condition_and_loop/vowel_consonant.py
# Write a Python program to check whether an alphabet is a vowel or consonant
vowels = ('a', 'e', 'i', 'o', 'u')


def vowel_or_consonant():
    s = input("Say a something: ")

    for i in s:
        print(check_what_is(i))

def check_what_is(s):
    is_a = ""
    if s in vowels:
        is_a = "vowel"
    else:
        is_a = "consonant"

    return s+" is a: "+is_a

vowel_or_consonant()
